phonebookapp/src/main/java/homeworks/adina/ContactsDb.java

```diff
@@ -0,0 +1,62 @@
+package homeworks.adina;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ContactsDb {
+
+
+    public static List<Person> readContacts() throws IOException {
+        File file = new File("phonebookapp/src/main/resources/adinaFiles/contactsDb.txt");
+
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(new FileInputStream(file)));
+        String line = bufferedReader.readLine();
+        List<Person> contactList = new ArrayList<Person>();
+
+        while (line != null) {
+            String[] contactArray = line.split(";");
+            String[] nameSurnameArray = contactArray[1].split(" ");
+
+            Person contact = new Person();
+            contact.setId(Integer.parseInt(contactArray[0]));
+            contact.setPhoneNumber(contactArray[2]);
+            contact.setFirstName(nameSurnameArray[0]);
+            if (nameSurnameArray.length > 1) {
+                contact.setLastName(nameSurnameArray[1]);
+            }
+            contactList.add(contact);
+            line = bufferedReader.readLine();
+
+        }
+        return contactList;
+
+    }
+
+    public static void saveNewContactList() {
+        File contactsFile = new File("phonebookapp/src/main/resources/adinaFiles/contactsDb.txt");
```

Comment:
Let us make this read it as an input to function. So that we can give user to change file.

Reply:
@gantu done